VM/rel2vm_params.py

```diff
@@ -881,92 +874,91 @@ def main(
     return vm_params_dict
 
 
-def load_msgs(args, logger: Logger = qclogging.get_basic_logger()):
+def load_msgs(
+        ds_multiplier,
+        dt,
+        hh,
+        min_vs,
+        no_optimise,
+        out_dir,
+        pgv,
+        rel_file,
+        vm_topo,
+        vm_version,
+        logger: Logger = qclogging.get_basic_logger()
+):
     # returns list of appropriate srf metadata
     msgs = []
-    faults = set()
-
-    # add task for every info file
-    rel_files = glob(args.rel_glob)
-    rel_files.sort()
-    # add task for every rel file
-
-    for rel in rel_files:
-        if not os.path.exists(rel):
-            logger.info("REL csv file not found: {}".format(rel))
-            continue
 
-        # name is unique and based on basename
-        name = os.path.splitext(os.path.basename(rel))[0].split("_")[0]
-
-        if name in faults:
-            continue  # this fault is already registered
-        logger.debug("Found first REL file for {}".format(name))
-        faults.add(name)
-
-        a = pd.read_csv(rel)
-        rake = a["rake"].loc[0]
-
-        planes = []
-        num_subfaults = a["plane_count"].loc[0]
-        for i in range(num_subfaults):
-            plane = {}
-            plane["centre"] = [
-                a["clon_subfault_{}".format(i)].loc[0],
-                a["clat_subfault_{}".format(i)].loc[0],
-            ]
-            plane["length"] = a["length_subfault_{}".format(i)].loc[0]
-            plane["width"] = a["width_subfault_{}".format(i)].loc[0]
-            plane["strike"] = a["strike_subfault_{}".format(i)].loc[0]
-            plane["dip"] = a["dip_subfault_{}".format(i)].loc[0]
-            plane["dtop"] = a["dtop_subfault_{}".format(i)].loc[0]
-            planes.append(plane)
-
-        corners, dbottom = get_corners_dbottom(planes, dip_dir=a["dip_dir"].loc[0])
-
-        mag = a["magnitude"].loc[0]
-        # try:
-        #    mag = mom2mag(sum(map(mag2mom, a["magnitude"].loc[0])))
-        # except TypeError:
-        #    mag = a["magnitude"].loc[0]
-        dtop = a["dtop"].loc[0]
-        dbottom = a["dbottom"].loc[0]
-
-        hdepth = (
-            np.round(a["dhypo"].loc[0], decimals=1)
-            * np.sin(np.radians(a["dip"].loc[0]))
-            + dtop
-        )
+    #rel_file is assumed to be the first realisation
```

Comment:
@jasonmotha  What do you mean by "median" here? REL02 out of REL01,REL02,REL03? @jpa198  Your comment here please?

Reply:
Rel01/02/03 is the realisations potentially with perturbated magnitude etc. There should be a median event file with unperturbated values (the characteristic Mw etc)

Reply:
This seems to be a variable naming issue, rather than an external file/type issue. As long as this script is given the right file path it should be fine.
This comment should be deleted, and the file existence check moved to load_args

Reply:
ok. this script will be happy as long as it gets a rel csv file. The orchestration script (to be developed) will decide which rel csv file to give to this script.